projects/ngx-openlayers/src/lib/layers/layerimage.component.ts

```diff
@@ -11,9 +11,7 @@ import { Extent } from 'ol/extent';
     <ng-content></ng-content>
   `,
 })
-export class LayerImageComponent extends LayerComponent implements OnInit, OnChanges {
-  public source: Image;
-
```

Comment:
The `source` field here and in several other layers was seemingly unused, and conflicted with the `source` field in the layer `Options` interface, so I removed them.